ipatests/test_integration/test_commands.py

```diff
@@ -1618,6 +1595,54 @@ def test_ipa_context_manager(self):
         result = host.run_command([paths.KLIST])
         assert host_princ in result.stdout_text
 
+    def test_delete_last_enabled_admin(self):
+        """
+        The admin user may be disabled. Don't allow all other
+        members of admins to be removed if the admin user is
+        disabled which would leave the install with no
+        usable admins users
+        """
+        user = 'adminuser2'
+        passwd = 'Secret123'
+        tasks.create_active_user(self.master, user, passwd)
+        tasks.kinit_admin(self.master)
+        self.master.run_command(['ipa', 'group-add-member', 'admins',
+                                '--users', user])
+        tasks.kinit_user(self.master, user, passwd)
+        self.master.run_command(['ipa', 'user-disable', 'admin'])
+        result = self.master.run_command(
+            ['ipa', 'user-disable', 'admin'],
```

Comment:
Did you mean to do instead ipa user-del user?

Reply:
sigh, yes. Fixed.